user-manual/ingest/ingest.rst

```diff
@@ -65,6 +65,41 @@ Create a SIP
 #. For selecting a normalization option, see :ref:`Normalization <normalize>`,
    below.
 
+The microservices that run on the ingest tab include:
+
+* **Verify SIP compliance**: verifies that the SIP conforms to the folder structure required by Archivematica for processing.
```

Comment:
Just a reminder about the 80 character limit! This (and the other bullet points here) should be changed from 

```
* **Verify SIP compliance**: verifies that the SIP conforms to the folder structure required by Archivematica for processing.
```

to

```
* **Verify SIP compliance**: verifies that the SIP conforms to the folder
  structure required by Archivematica for processing. 
```

(Note that the second line is indented two spaces to line up with the beginning of list item.)